### DISCUSSION TITLE
StateTimeline: Option to hide series name

### DISCUSSION THREAD

**Discussion by pepjo** [grafana/grafana#38316] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Hide the series name that appears on the left.

![image](https://user-images.githubusercontent.com/1307305/123619397-0a0b1f00-d801-11eb-9717-63916845e6da.png)
In this screenshot, the name is `e`.

**Why is this needed**:
When there is only a single series on a pannel, the name is not needed.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
